src/server/main.go

```diff
@@ -31,6 +32,27 @@ type ServerConfig struct {
 	RedisPort         string
 	WhitelistPath     string
 	DevMode           bool
+	LogLevel          string
+}
+
+func parseLogLevel(level string) slog.Level {
+	if level == "" {
+		return slog.LevelInfo
+	}
```

Comment:
Regardless of if we implement the inline or cadence.env method to implement log levels, I do think this should default to debug mode, for the reasons outlined in the other comment above for practicality of an everyday cadence user sending logs. Open to your feedback.

Reply:
I'm not picky about the default, so if you have a preference towards `debug` that's fine.

I tend to lean towards having a level below the level that users get though - Users get enough information to follow what's going on and see when there is a problem, and devs get enough information from that (hopefully) to be able to replicate it and observe more - Ideally.

I think that's the only real reason one would have the default above the lowest level - To avoid spamming the user, and help them highlight when issues are happening (It's easier for us to understand an issue that describes "I start the server and I get this message in the log saying songs aren't in the database even though it got populated, and the UI won't connect" than just "My UI isn't working" with a massive log attached to it).

Reply:
I see your reasoning so I'm good with giving `info` as a default.

Reply:
Cadence is reasonably straightforward and stable at this point so the majority of users don't need to see every debug action happening, which are for the most part an element of the developer experience.